src/io/console.cr

```diff
@@ -22,7 +22,7 @@ class IO::FileDescriptor
   # Only call this when this IO is a TTY, such as a not redirected stdin.
   def noecho!
     if LibC.tcgetattr(fd, out mode) != 0
-      raise Errno.new "can't set IO#noecho!"
+      raise OSError.create "can't set IO#noecho!"
```

Comment:
Could we have exception message capitalized? I know it's out of the scope of this PR but since it's a line change anyway... ;)

Reply:
Definitely out of scope. And there are other identical downcase messages in the same module that would also need to be changed.
In fact, please remove these suggestions to avoid noise 😐

Reply:
I see your point, sorry for the noise 👍